bin/start_backend.sh

```diff
@@ -187,12 +187,24 @@ if [ -f $pidfile ]; then
     # check if the binary name can be grepped from the process cmdline.
     # still it has chances to be false positive, but the possibility is greatly reduced.
     oldpid=$(cat $pidfile)
-    pscmd=$(ps -q $oldpid -o cmd=)
-    if echo "$pscmd" | grep -q -w "$BIN_NAME" &>/dev/null ; then
-        echo "Backend running as process $oldpid. Stop it first."
-        exit 1
+    if ps -p $oldpid > /dev/null 2>&1; then
+        psstatus=$(ps -q $oldpid -o stat=)
+        if [[ "$psstatus" == *Z* ]]; then
```

Comment:
why bother killing the process, just remove the pid and start the process.

Reply:
If you don't deal with the old zombie processes, the zombie processes will always be there. And I don't think it's correct to print that the process already exists and then exit because a zombie process exists.

Reply:
I mean you can just ignore the zombie process and start the new process.

Reply:
@xyllq999 please let me know if you have any thoughts on this.